src/main/kotlin/racingcar/Game.kt

```diff
@@ -0,0 +1,20 @@
+package racingcar
+
+class Game(private val carNumber: Int) {
+    private val carList = createCarList(carNumber)
+
+    private fun createCarList(carNUmber: Int): ArrayList<RacingCar> {
+        val returnList = arrayListOf<RacingCar>()
+        for (number in 1..carNUmber) {
+            returnList.add(RacingCar(number.toString()))
+        }
+        return returnList
+    }
+
+    fun playGameOnce(): ArrayList<RacingCar> {
+        carList.forEach {
+            it.randomMoving()
+        }
+        return carList
```

Comment:
onEach 메소드가 있었던 걸로 기억하는데 활용할 수 있지 않을까 싶네요.
저도 메소드의 구현을 살펴보진 않았는데 적용 가능한지 한번 확인해보세요~

```suggestion
        return carList.onEach {
            it.randomMoving()
        }
```